imap_processing/cli.py

```diff
@@ -45,11 +52,27 @@ def _parse_args():
         "The data level to process. Acceptable values are: "
         f"{imap_processing.PROCESSING_LEVELS}"
     )
+    depdency_help = (
+        "Dependency information in str format."
+        "Example: '[{instrument: swe, level: l0, version: v00-01}]'"
+    )
 
     parser = argparse.ArgumentParser(prog="imap_cli", description=description)
-    parser.add_argument("--data-dir", type=str, required=False, help=data_dir_help)
     parser.add_argument("--instrument", type=str, required=True, help=instrument_help)
     parser.add_argument("--level", type=str, required=True, help=level_help)
+    parser.add_argument(
+        "--file_path",
```

Comment:
If we can also change the event kickoff, we should try and stick to hyphens only in the CLI arguments.
```suggestion
        "--file-path",
```